exp-player/addon/templates/components/exp-video-consent.hbs

```diff
@@ -14,29 +22,57 @@
       </div>
     </div>
   </div>
+  </div>
+
+
+  <div class="col-md-5">
+
+    <div class="video-consent-numbered-direction">
+    <span class="video-consent-number">2.</span>
+    <span class="video-consent-direction">Click to <button class="center" disabled={{disableRecord}} class="btn btn-primary pull-left" {{ action 'record' }}>start consent recording...</button></span>
+
+  </div>
 
+      <div class="row recorder-container" style="height: 30vh; margin: 0px; ">
+      <div class="recorder col-xs-12 col-md-12" style="height:100%"></div>
+    </div>
+
+  <div class="video-consent-numbered-direction">
+    <span class="video-consent-number">3.</span>
+    <span class="video-consent-direction">Read the statement below <em class="video-consent-outloud"> out loud: </em></span>
+  </div>
 
-  <div class="col-md-4">
+  <!--
     <blockquote>
       <p>
         If you're ready to participate, please make a consent recording to get started! Press "record" and then read the statement below (in green) out loud.
       </p>
-    </blockquote>
+    </blockquote>-->
     <div class="well video-consent-prompt">
       <span class="breaklines exp-video-consent-prompt">{{prompt}}</span>
     </div>
 
-    <div class="row recorder-container" style="height: 30vh; margin: 5px; ">
-      <div class="recorder col-xs-12 col-md-12" style="height:100%"></div>
-    </div>
 
-      <div class="row video-consent-controls">
-	  {{#if recordingStarted }}
-	      <button class="btn btn-success pull-right" {{ action 'finish' }}>Finish</button>
+
+
+
+  <div class="video-consent-numbered-direction">
+    <span class="video-consent-number">4.</span>
+    <span class="video-consent-direction">Click to
+    	  {{#if recordingStarted }}
+	      <button class="btn btn-success pull-right" {{ action 'finish' }}>submit</button>
 	  {{else}}
-	      <button disabled={{disableRecord}} class="btn btn-primary pull-left" {{ action 'record' }}>Record</button>
+	      <button disabled="disabled" class="btn btn-success pull-right" {{ action 'finish' }}>submit!</button>
```

Comment:
Note for later: one button has an exclamation point but the other does not; this might be confusing UI.